encoding/protobuf/protoc-gen-yarpc-go/internal/testing/testing.pb.yarpc.go

```diff
@@ -237,23 +242,549 @@ func (h *_SinkYARPCHandler) Fire(ctx context.Context, requestMessage proto.Messa
+func newSinkServiceFireYARPCResponse() proto.Message {
+	return &yarpcproto.Oneway{}
+}
+
+var (
+	emptySinkServiceFireYARPCRequest  = &FireRequest{}
+	emptySinkServiceFireYARPCResponse = &yarpcproto.Oneway{}
+)
+
+// AllYARPCClient is the YARPC client-side interface for the All service.
+type AllYARPCClient interface {
+	GetValue(context.Context, *GetValueRequest, ...yarpc.CallOption) (*GetValueResponse, error)
+	SetValue(context.Context, *SetValueRequest, ...yarpc.CallOption) (*SetValueResponse, error)
+	Fire(context.Context, *FireRequest, ...yarpc.CallOption) (yarpc.Ack, error)
+	HelloOne(context.Context, ...yarpc.CallOption) (AllServiceHelloOneYARPCClient, error)
+	HelloTwo(context.Context, *HelloRequest, ...yarpc.CallOption) (AllServiceHelloTwoYARPCClient, error)
+	HelloThree(context.Context, ...yarpc.CallOption) (AllServiceHelloThreeYARPCClient, error)
+}
+
+// AllServiceHelloOneYARPCClient sends HelloRequests and receives the single HelloResponse when sending is done.
+type AllServiceHelloOneYARPCClient interface {
+	Context() context.Context
+	Request() *transport.StreamRequest
+	Response() *transport.StreamResponse
+	Send(*HelloRequest) error
+	CloseAndRecv() (*HelloResponse, error)
+}
+
+// AllServiceHelloTwoYARPCClient receives HelloResponses, returning io.EOF when the stream is complete.
+type AllServiceHelloTwoYARPCClient interface {
+	Context() context.Context
+	Request() *transport.StreamRequest
+	Response() *transport.StreamResponse
+	Recv() (*HelloResponse, error)
+}
+
+// AllServiceHelloThreeYARPCClient sends HelloRequests and receives HelloResponses, returning io.EOF when the stream is complete.
+type AllServiceHelloThreeYARPCClient interface {
+	Context() context.Context
+	Request() *transport.StreamRequest
+	Response() *transport.StreamResponse
+	Send(*HelloRequest) error
+	Recv() (*HelloResponse, error)
+	CloseSend() error
+}
+
+// NewAllYARPCClient builds a new YARPC client for the All service.
+func NewAllYARPCClient(clientConfig transport.ClientConfig, options ...protobuf.ClientOption) AllYARPCClient {
+	return &_AllYARPCCaller{protobuf.NewStreamClient(
+		protobuf.ClientParams{
+			ServiceName:  "uber.yarpc.encoding.protobuf.protocgenyarpcgo.internal.testing.All",
+			ClientConfig: clientConfig,
+			Options:      options,
+		},
+	)}
+}
+
+// AllYARPCServer is the YARPC server-side interface for the All service.
+type AllYARPCServer interface {
+	GetValue(context.Context, *GetValueRequest) (*GetValueResponse, error)
+	SetValue(context.Context, *SetValueRequest) (*SetValueResponse, error)
+	Fire(context.Context, *FireRequest) error
+	HelloOne(AllServiceHelloOneYARPCServer) (*HelloResponse, error)
+	HelloTwo(*HelloRequest, AllServiceHelloTwoYARPCServer) error
+	HelloThree(AllServiceHelloThreeYARPCServer) error
+}
+
+// AllServiceHelloOneYARPCServer receives HelloRequests.
+type AllServiceHelloOneYARPCServer interface {
+	Context() context.Context
+	Request() *transport.StreamRequest
+	SetResponse(*transport.StreamResponse)
```

Comment:
Same concern as above. Metadata for inbound calls is/should be on the context already.